tests/resources/klargjorte_data/person_testdata_p2021-12-31_p2021-12-31_v1__DOC.json

```diff
@@ -52,11 +52,7 @@
                 "nb": "Norge"
             },
             "id": "2f72477a-f051-43ee-bf8b-0d8f47b5e0a7",
-            "owner": {
```

Comment:
What happens if a user loads existing metadata with this structure? Do we handle it gracefully or does it fail? It's important to think about backwards compatibility with model updates. If it fails we should make an upgrade function in `model_backwards_compatibility.py`